src/native/common.h

```diff
@@ -0,0 +1,61 @@
+// Straight from node source code https://github.com/nodejs/node/blob/master/test/addons-napi/common.h
```

Comment:
Forgive me, I'm not too familiar with NAPI–is it expected that all NAPI projects include a copy of this file? Seems like something the node runtime should provide.

Reply:
That was my initial thought, but I got it as not defined